src/utils/powersystems_utils.jl

```diff
@@ -360,3 +360,7 @@ function get_deterministic_time_series_type(sys::PSY.System)
         )
     end
 end
+
+
+"""Overload get_variable for MarketBidCost. Returns nothing"""
+PSY.get_variable(value::PSY.MarketBidCost) = nothing
```

Comment:
What is this for? If it is needed, it should be put in PowerSystems. See [here](https://docs.julialang.org/en/v1/manual/style-guide/#avoid-type-piracy) on "type piracy," better definition [here](https://discourse.julialang.org/t/how-bad-is-type-piracy-actually/37913/6).

Reply:
This shouldn't be used at all. The implementation of MarketBidCost needs to be unique and avoid this call. Moreover, this is a legacy call to older implementations